src/upload/system/library/transbank/Exceptions/TransactionCreationException.php

```diff
@@ -0,0 +1,9 @@
+<?php
+namespace Transbank\Exceptions;
+class TransactionCreationException extends \Exception
```

Comment:
use infinitive in this name `TransactionCreateException`. In other places, it is used this way